packages/headless/src/features/pagination/pagination-analytics-actions.ts

```diff
@@ -35,3 +37,19 @@ export const logPagePrevious = (): LegacySearchAction =>
       pagerNumber: currentPageSelector(state as PaginationSection),
     })
   );
+
```

Comment:
```suggestion
// TODO KIT-2983
```

We're gonna change'em